scripts/network.lua

```diff
@@ -72,6 +72,9 @@ function Network.tick()
       -- Check for items ready on each input port lane
       local canDrop = false
       for idx = 1, 2 do
+        if not entity or not entity.valid then
```

Comment:
I ran into this issue also a few times but never could come up with a solid repro. Is this defensive or do you know a way to trigger the problem?

Reply:
Honestly can't remember exactly, but I crashed here once and added it defensively. It's very possible the crash was due to other things I was changing mid cycle on this PR though and this may not be truly necessary.